src/nidmm/system_tests/test_system_nidmm.py

```diff
@@ -94,28 +94,50 @@ def test_Enum_attribute(device_info):
         assert session.function == nidmm.Function.AC_CURRENT
         assert type(session.function) is nidmm.Function
         try:
-            session.function = nidmm.LCCalculationModel.CALC_MODEL_SERIES
+            session.function = nidmm.LCCalculationModel.SERIES
             assert false
         except TypeError as e:
             print(e)
             pass
 
 
-def test_ViSession_attribute(device_info):
+def test_acquisition(device_info):
+    with nidmm.Session(device_info['name']) as session:
+        session.configure_measurement_digits(nidmm.Function.DC_CURRENT, 1, 5.5)
+        with session.initiate():
+            print(session.fetch(1000))
+        with session.initiate():
+            print(session.fetch(1000))
+
+			
+def test_method_call_with_zero_parameter(device_info):
+    with nidmm.Session(device_info['name']) as session:
+        assert session.get_aperture_time_info()[1] == 0 # Assuming default aperture time unit will be seconds
+   
+        
+def test_method_call_with_one_parameter(device_info):
+    session.configure_power_line_frequency(60)
+
+        
+def test_invalid_method_call(device_info):
+    #calling a function, without parameter, But it has a mandate parameter
     with nidmm.Session(device_info['name']) as session:
         try:
-            session.io_session = 5
-            assert false
-        except TypeError as e:
-            print(e)
-            pass
-        try:
-            value = session.io_session
-            assert false
+            session.fetch()
```

Comment:
This would raise an exception even if fetch() didn't take a parameter because the session is not in the running state. Use a different function please.
Also, fetch() or read() are bad examples because we are planning on having a default value for the timeout parameter.